code/game/machinery/status_display.dm

```diff
@@ -90,6 +91,32 @@
 
 	update_appearance()
 
+/// If the display is displaying an alert level update it.
+/obj/machinery/status_display/proc/update_security_level()
+	SIGNAL_HANDLER
+
+	if(!(current_picture in GLOB.status_display_alert_level_pictures))
+		return
+	switch(SSsecurity_level.get_current_level_as_number())
+		if(SEC_LEVEL_DELTA)
+			set_picture("deltaalert")
+		if(SEC_LEVEL_RED)
+			set_picture("redalert")
+		if(SEC_LEVEL_BLUE)
+			set_picture("bluealert")
+		if(SEC_LEVEL_GREEN)
+			set_picture("greenalert")
+		if(SEC_LEVEL_AMBER)
+			set_picture("amberalert")
+		if(SEC_LEVEL_YELLOW)
+			set_picture("yellowalert")
+		if(SEC_LEVEL_LAMBDA)
+			set_picture("lambdaalert")
+		if(SEC_LEVEL_GAMMA)
+			set_picture("gammaalert")
+		if(SEC_LEVEL_EPSILON)
+			set_picture("epsilonalert")
```

Comment:
should prolly be a part of `/datum/security_level`

(again, won't be picky, still okay with merging this PR without that, but would be preferred if possible)